src/hooks/useWalletCloudBackup.js

```diff
@@ -26,6 +27,27 @@ export default function useWalletCloudBackup() {
       password,
       walletId,
     }) => {
+      const isAvailable = await isCloudBackupAvailable();
+      if (!isAvailable) {
+        Alert.alert(
+          'iCloud Error',
+          'Looks like iCloud drive is not enabled in your device. Do you want to see how to enable it?',
```

Comment:
I would use "on" your device instead of "in" your device.

Reply:
Would it be difficult to create our own version of the mobile-specific image/ instructions they have instead of redirecting them? I feel like linking them is more likely to be overwhelming and less likely for them to enable

Reply:
It's not, but let's go with this for now and let's ping @christianbaroni  in case he wants to design a modal with a screen shot of the apple website inside. 

OR we can also open a webview without you leaving the app.

Reply:
Sounds good! In that case, we can merge this in as is, and he can update copy / decide on what he'd like for instructions!